test/views/declaration/SubmissionConfirmationPageViewSpec.scala

```diff
@@ -58,16 +58,21 @@ class SubmissionConfirmationPageViewSpec extends UnitViewSpec with BeforeAndAfte
     super.afterEach()
   }
 
-  "Confirmation Page View on empty page" should {
+  "Confirmation Page View always" should {
 
-    "display header with default" in {
-      val highlightBox = getHighlightBox(createView())
-      highlightBox must containText("Declaration has been submitted")
-      highlightBox mustNot containText("The LRN is")
+    "display start again link" in {
+      val button = createView().getElementById("back-to-start-link")
+      button must haveHref(controllers.routes.ChoiceController.displayPage().url)
+      button must containText("Create a new declaration")
```

Comment:
Nit: Any reason to not use `containMessage("declaration.confirmation.submitAnotherDeclaration")` ?

Reply:
No, good call :+1: